lib/mockdata/mock_contact.dart

```diff
@@ -3,15 +3,20 @@ class Contact{
     String firstName; 
     String lastName; 
     String phoneNumber;
-
-    Contact({required this.id, required this.firstName, required this.lastName, required this.phoneNumber}); 
+    int gender; 
+    String email; 
+    String address; 
+    Contact({required this.id, required this.firstName, required this.lastName, required this.phoneNumber, 
+            this.address = "", this.email =" ", this.gender = 0}); 
 
     @override 
     String toString() {
         return("id: $id\nFirst name: $firstName\nLast name: $lastName\nTel: $phoneNumber"); 
   }
 }
 class MockContact{
-    static List<Contact> mocks() => 
```

Comment:
function name should be mocks which will return a List<> and mock which will return a single object.